### ISSUE TITLE
examplify compatibility with macOS

### ISSUE THREAD

**Issue by grizzzle** [nus-cs2030/2324-s2#771]
hi! i recalled seeing an email about macOS sonoma not being supported by examplify sometime last year. does anyone know if that issue has been resolved so i can upgrade my software if necessary?

**Comment by kongshiting** [comment]
hi i received an email yesterday from examsoft on this. 

---

With final exams around the corner, please check the OS version (https://wiki.nus.edu.sg/x/jABEF) on your exam device and update if necessary. No action is required if your current OS is supported.

1. Windows: Windows 10 22H2, Windows 11 22H2, and Windows 11 23H2
2. Mac: Monterey, Ventura, and Sonoma
3. iPad: iPadOS 15, iPadOS 16, and iPadOS 17

Please be informed that Examplify 3.4.6 (Windows) and 3.4.4 (Mac) are the versions officially supported by NUS. Kindly check and ensure that you are using the correct version of Examplify before your exam. This is important as using the wrong version may result in technical issues during the exam. The Examplify version is displayed at the bottom right corner of the Examplify dashboard.

---

i personally have not upgraded but you can check your email for more details and test out the mock assessment prof uploaded on examplify. all the best for finals!

**Comment by starryfieldz** [comment]
Yup Sonoma was not compatible during last sem, but it is now compatible already. If you havent update, I suggest not updating until after finals just in case anything goes wrong.

**Comment by azzy83** [comment]
Yeah, last sememster i made the mistake of upgrading to Sonoma ( but strangely all my exams on examplfiy went well, no issues except for slow loading time and occasional freezes). But yeah, as another user suggested, don't update anything till after the paper, only a few more days anyways.

**Comment by CSnoobCS** [comment]
Try out the practice paper on examplify and see if your examplify works. I all's good then dont do anything lol

**Comment by lohkeyyy** [comment]
thank you

**Comment by grizzzle** [comment]
okay! thank you all

**Comment by ZHD1987E** [comment]
finals is over you can close this issue
